source/extensions/omni.isaac.lab/omni/isaac/lab/managers/observation_manager.py

```diff
@@ -337,6 +337,23 @@ def _prepare_terms(self):
                 obs_dims = tuple(term_cfg.func(self._env, **term_cfg.params).shape)
                 self._group_obs_term_dim[group_name].append(obs_dims[1:])
 
+                # if scale is set, check if single float or tuple
+                if term_cfg.scale is not None:
+                    if not isinstance(term_cfg.scale, (float, int, tuple)):
+                        raise TypeError(
+                            f"Scale for observation term '{term_name}' in group '{group_name}'"
+                            f" is not of type float, int or tuple. Received: '{type(term_cfg.scale)}'."
+                        )
+                    if isinstance(term_cfg.scale, tuple) and len(term_cfg.scale) != obs_dims[1]:
+                        raise ValueError(
+                            f"Scale for observation term '{term_name}' in group '{group_name}'"
+                            f" does not match the dimensions of the observation. Expected: {obs_dims[1] - 1}"
+                            f" but received: {len(term_cfg.scale)}."
+                        )
+
+                    if isinstance(term_cfg.scale, tuple):
```

Comment:
We can always make it a tensor. Shouldn't affect the broadcasting and might be slightly faster since it a tensor?

Reply:
yes that is better, will change it